### PR TITLE
Feat : added snackbar component

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
* Added Snackbar component
* Deploy link : https://deploy-preview-12--fast-ui.netlify.app/

For screenshot see the next comment. :)

### CODE REVIEW

**Comment by Mohasin-Haque** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /pages/toast.html
Browser:   Chrome 97.0.4692.99 on Windows 10
Viewport:  1366 x 649 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-12--fast-ui.netlify.app%2Fpages%2Ftoast.html&os=Windows&os_version=10&browser=Chrome&browser_version=97.0&resolution=1366x649&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-12--fast-ui.netlify.app/pages/toast.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61feea55f76b1f0007be7ec6&commentID=61feead2b19bdab6b1bd71b2&resolution=resolved)

File: pages/toast.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <!-- Leading -->
+            <div class="toast-container">
+                <div class="toast-text">Your photo has been archieved.</div>
+                <div class="retry undo">UNDO</div>
+                <div class="toast-icon">X</div>
+            </div>
+            </div>
+
+
+
```
Comment (devdeadviz) [inline pages/toast.html]: Remove extra spaces.

**Review by devdeadviz — COMMENTED** [review]
Looks good to me, just need to do some code formatting.

File: styles/toast.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.toast-container{
+    background-color: var(--text-color-black);
+    color: var(--body-background-color);
+    display: flex;
+    flex-direction: row;
```
Comment (xii-vi) [inline styles/toast.css:5]: flex-direction is by default row

File: styles/toast.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    cursor: pointer;
+    margin-right: 1rem;
+}
+
+.retry:hover{
+    color: var(--hover-color);
+}
+
+.toast-icon{
+    cursor: pointer;
```
Comment (xii-vi) [inline styles/toast.css:27]: make cursor utility class and use whereever required

**Review by xii-vi — COMMENTED** [review]
Work on the suggested changes and then you can merge.

File: styles/toast.css (hunk 2 above)
Comment (Mohasin-Haque) [reply styles/toast.css:5]: in this case i have used flex center property where i have given the flex direction as column so did here

File: styles/toast.css (hunk 3 above)
Comment (Mohasin-Haque) [reply styles/toast.css:27]: ok

**Pull request merged**